proposals/052-cryptography-leg-1.md

```diff
@@ -0,0 +1,329 @@
+
+### Intermediate deliverables
+
+The following are intermediate deliverables intended to keep the community apprised of any progress that is made, and any challenges that arise, and to seek community feedback:
+
+- Frequent (2-3x weekly) activity update on devlog
+- Weekly major update that includes github push
+- Monthly report that summarizes what specific items have been worked on, what items will be worked on, and what challenges have arisen, if any.
+
+### Deliverables
+
+The following are the intended deliverables of this leg of the proposal:
+
+- `botan-bindings`
+- `botan-low`
+- `botan`
+
+Each library should be beta- or / production-ready, and submitted to hackage as a candidate or accepted package. 
+
+Each library should have:
+
+- Documentation
+- Tutorials and examples
+- Tracked issues
+- Unit tests
+
+A project website will also be created.
+
+The official repo will be owned by the Haskell Cryptography Group, and another member of the Haskell Cryptography Group will be selected to hold backup keys and permissions to the website, server, and official repositories as needed, in order to give project members access and to reduce any conflicts about ownership or maintenance.
+
+## Budget
+
+The first leg of this proposal presents a minimum budget of $7000 USD per month, for one full-time engineer, for a duration of 3 months, for a total of $21,000 USD. This budget is based on cost-of-living and industry experience, and will cover housing, food, bills, taxes, and other life necessities for one engineer, as well as any project necessities such website and server hosting. This budget is roughly equivalent to $40 / hr or $84k / yr at full-time of 40 hrs / wk. Industry rates for an engineer of the necessary skill are on average considerably higher, and so we consider this budget to be reasonable. The exact legal contract / arrangement is left to the Haskell Foundation.
+
+The question of continued funding will be revisited for each future leg of this proposal, wherein the budget and direction of future work will be decided again for each leg.
+
+## Stakeholders
+
+Several parties stand to gain from the implementation of this proposal.
+
+- Leon Dillinger (Myself)
+
+As the intended engineer carrying out the work, I have already invested significant effort in this project in order to establish its viability, and will benefit from this proposal in the form of being the recipient of the funding.
+
+- [Haskell Foundation](https://haskell.foundation/)
+
+As the intended source of funding, the Haskell Foundation would be invested in this proposal financially, and its effective success or failure would reflect respectively upon the Haskell Foundation.
+
+- [Haskell Cryptography Group ](https://github.com/haskell-cryptography)
+
+As the intended owner of the project, this project will be new maintenance burden on the Haskell Cryptography Group.
```

Comment:
Do we think they have the resources to take care of this? If so, it would be good to say so here.

Reply:
I am a member of the cryptography group, so I am the intended recipient of this maintenance burden - I should definitely clarify this.